### DISCUSSION TITLE
Adding strictMessage: false to VueI18nPlugin in quasar.config.js

### DISCUSSION THREAD

**Discussion by anli-xsigns** [quasarframework/quasar#15824] [CLI - PWA mode] [answered]
Hello everyone,

according to https://stackoverflow.com/questions/75922094/how-can-i-use-vue-i18n-in-vite-to-translate-html I have to add 

```
plugins: [
  VueI18nPlugin({
    ...
    strictMessage: false
  }),
...
]
```
to my vite config. How can I achieve this via quasar.config.js -> build -> extendViteConf as described in https://quasar.dev/quasar-cli-vite/handling-vite/? Any help is highly appreciated. Thanks a lot in advance!

**Comment by alexrififi** [comment, marked as answer]
```js
// quasar.config.js

vitePlugins: [
  [
    '@intlify/vite-plugin-vue-i18n',
    {
      strictMessage: false,
    },
  ],
]
```
